src/components/buttons/primary-action-button/primary-action-button.story.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { withKnobs, boolean, text, select } from '@storybook/addon-knobs';
+import withReadme from 'storybook-readme/with-readme';
+import Section from '../../../../.storybook/decorators/section';
+import * as icons from '../../icons';
+import PrimaryActionButton from './primary-action-button';
+import Readme from './README.md';
+
+const iconNames = Object.keys(icons);
+
+storiesOf('Buttons', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withReadme(Readme))
+  .add('1. Primary Action Button', () => (
```

Comment:
is the `1. ` intentional

Reply:
Yes. The idea is to organize the buttons better. In the end with something like:

![image](https://user-images.githubusercontent.com/14938754/45222728-939a7c80-b2b5-11e8-8a36-5f199e6238a6.png)

Reply:
I don't like the numbering. There is no real order of them.